libs/utils/env.py

```diff
@@ -441,6 +448,8 @@ def _init_target(self, force = False):
 
         elif self.conf['board'] != 'UNKNOWN':
             # Initilize from platform descriptor (if available)
+            self._log.error('Warning: UNKNOWN board type detected!!')
```

Comment:
I don't disagree with the code, I just think that moving these few lines in a separate commit would make the history cleaner.